cmd/analyzer/analyzer.go

```diff
@@ -22,7 +23,26 @@ import (
 type BuildFlags struct {
 	Version string
 }
+
+type MistOptional struct {
+	Name   string `json:"name"`
+	Help   string `json:"help"`
+	Option string `json:"option,omitempty"`
+	// Short   string `json:"short"`
+	Default string `json:"default,omitempty"`
+}
+
+type MistConfig struct {
+	Name         string         `json:"name"`
+	Description  string         `json:"desc"`
+	FriendlyName string         `json:"friendly"`
+	Optional     []MistOptional `json:"optional,omitempty"`
+	Version      string         `json:"version,omitempty"`
+}
+
 type cliFlags struct {
+	json bool
```

Comment:
WDYT of calling this `mistJson` or `mistJsonHelp`, just so it's clearer what this means in the code? We can keep the same `-j` flag externally anyway, since I guess `Mist` requires it to be that.

Reply:
updated this. thanks :)